src/BaselineUtils.cc

```diff
@@ -1037,8 +1129,16 @@ symmetrizeFootprint(
         if (dxlo <= dxhi) {
             LOGL_DEBUG(_log, "Adding span fwd %i, [%i, %i],  back %i, [%i, %i]",
                        fy, cx+dxlo, cx+dxhi, by, cx-dxhi, cx-dxlo);
-            sfoot->addSpan(fy, cx + dxlo, cx + dxhi);
-            sfoot->addSpan(by, cx - dxhi, cx - dxlo);
+            if (cx + dxlo <= cx + dxhi) {
+                tmpSpans.push_back(geom::Span(fy, cx + dxlo, cx + dxhi));
+            } else {
+                tmpSpans.push_back(geom::Span(fy, cx + dxhi, cx + dxlo));
+            }
```

Comment:
The old code had: sfoot->addSpan(fy, cx + dxlo, cx + dxhi); in it. addSpans had a check on which was lower and would swap the arguments if they were entered wrong (essentially the same if statements moved here) I was trying to ensure that the behavior of addSpans stayed exactly the same. cx is a signed int, so theoretically it could be negative (a wierd xy0 for the image). This could cause what is lower and higher to flip (though now that I think about it longer, only the the case for cx -dx(lo/hi)). This check could probably be removed for the addition, but I would want someone to verify thatI am right in that, as it would be removing a check we used to have.